requirements/main.txt

```diff
@@ -4,7 +4,7 @@ ffmpeg
 hydra-core
 joblib
 librosa>=0.10.0 # specify >=0.10.0 so that librosa.get_duration(path=...) will work
-numpy
+numpy==1.26
```

Comment:
Why is it fixed?

Reply:
Otherwise it installs numpy==2.0.0, which is incompatible with most of the nemo calls